pytest_pootle/plugin.py

```diff
@@ -55,3 +63,9 @@ def pytest_configure(config):
             fixtures_fs))
     for plugin in pytest_plugins:
         config.pluginmanager.import_plugin(plugin)
+
+
+def pytest_runtest_setup(item):
+    marker = item.get_marker("pootle_memusage")
+    if marker is not None and not item.config.getoption("--memusage"):
```

Comment:
```
os.environ["TRAVIS_EVENT_TYPE"] != "cron"
```
Would allow execution of these tests if we're running under Travis cron.  Just not sure that's the best way to configure this.